### DISCUSSION TITLE
How pin 'Select' column?

### DISCUSSION THREAD

**Discussion by willthebrit** [KevinVandy/mantine-react-table#450] [Q&A]
I am able to pin columns easily with `initialState={{ columnPinning: { left: ["name"] } }}`

I am unable to pin the 'Select' column with` initialState={{ columnPinning: { left: ["select"] } }}`

How can I left pin the select column? thank you.

![Screenshot 2024-11-26 at 10 20 17 AM](https://github.com/user-attachments/assets/3738e748-6e0b-4821-b164-52688c7a0c47)

**Comment by WojakGra** [comment]
I know, old question but shouldn't it be `mrt-row-select`?
